_ext/repository.rb

```diff
@@ -185,11 +185,9 @@ def execute(site)
                   r.commits_url = repo_data.first['commits_url']
                 end
               }
-              if org_repos_data.headers[:link] =~ /<(.*)>.+rel="next",/
-                org_url = $1
-              else
-                more_pages = false
-              end
+              links = parse_link_headers(org_repos_data)
+              org_url = links[:next]
+              more_pages = (links.has_key?(:next) and links.has_key?(:last) and (links[:next] != links[:last]))
```

Comment:
why does the next link cannot equal to the last link? I guess that this should be the last page (link) we should check. 
I believe that checking for the presence of the next link should be enough:
`more_pages = links.has_key?(:next)`

Reply:
There is always `next` and that was the problem of infinite loop.

Reply:
I was wrong (let's blame lack of coffee ;)). Thanks, it's fixed.